rhasspysupervisor/__init__.py

```diff
@@ -2155,12 +2155,8 @@ def get_text_to_speech(
 
     if tts_system == "wavenet":
 
-        voice = str(profile.get("text_to_speech.wavenet.voice", "Wavenet-C")).strip()
-        gender = str(profile.get("text_to_speech.wavenet.gender", "FEMALE")).strip()
+        voice = str(profile.get("text_to_speech.wavenet.voice", "en-US-Wavenet-C")).strip()
```

Comment:
apparently this line should be split, since the current line length limit is set to 88.
@koenvervloesem whats the reason for this line length limit? Following Pep8 it would be 80, but I guess something like 100 or 120 would be expected aswell. However 88 is rather specific and no limit I would have expected :sweat_smile:

Reply:
We're using 88 because [that's what Black uses](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length):

> You probably noticed the peculiar default line length. Black defaults to 88 characters per line, which happens to be 10% over 80. This number was found to produce significantly shorter files than sticking with 80 (the most popular), or even 79 (used by the standard library). In general, 90-ish seems like the wise choice.

Reply:
Ah ok I have never seen the limit of 88 before :)

Reply:
By the way, a `make reformat` should solve this issue automatically :-)

Reply:
Thanks, I was not aware. I will fix it asap.

Reply:
Was my reformatting ok or does it need more rework?

Reply:
Its fine now :smile: